src/docs/release-notes/R16/R16.mdx

```diff
@@ -0,0 +1,23 @@
+### Documentation Site
+
+Added search capability to doc site
+
+### New Components
+
+We have added new components to our [Angular](https://pxblue-components.github.io/angular) and [React Native](https://pxblue-components.github.io/react-native) component libraries:
+
+-   For Angular, we have added components for [Drawer](https://pxblue-components.github.io/angular-dev/?path=/info/components-drawer--readme), [ListItemTag](https://pxblue-components.github.io/angular-dev/?path=/info/components-list-item-tag--readme), [Spacer](https://pxblue-components.github.io/angular-dev/?path=/info/components-spacer--readme), and
```

Comment:
In our documentation we're referring to these as `<pxb-drawer>` etc. Do we want to do that here, or stick with the TitleCase?

Reply:
I think for our release notes we should stick to TitleCase since that's what our earlier releases have been using and it looks fine to me.